cal/cal/static/js/app.js

```diff
@@ -223,9 +227,17 @@ function TagsDetailCtrl($scope, $interpolate, $http, QueryService) {
     });
   };
 
-  this.initialize = function() {
-    this.showWeekly();
-    $http({method: 'GET', url: tagUrl + '.json' }).
+  this.showPageEvents = function() {
+    var start = this.currentPage * this.pageSize;
+    var end = (start + this.pageSize > this.tagEvents.length) ? this.tagEvents.length : start + this.pageSize;
+    this.pageEvents = [];
+    for (var i = start; i < end; i++) {
+      this.pageEvents.push(_this.tagEvents[i]);
+    }
+  }.bind(this);
+
+  this.getEvents = function(pageNum) {
+    $http({method: 'GET', url: tagUrl + '.json', params:{page:pageNum}}).
```

Comment:
You should install a linter - I'm getting yelled at here. Spacing should be `params: { page: pageNum }}`